### PR TITLE
[Mean circle] Instructions on how to use

### PR DESCRIPTION
**Pull request by carlowiesse**

### CODE REVIEW

File: mcr_object_recognition_mean_circle/README.md [hunk 1]
```diff
@@ -1 +1,32 @@
 The current model is trained with sklearn version 0.14.1.
+
+### How to use
```
Comment (minhnh) [inline mcr_object_recognition_mean_circle/README.md]: Fewer `#` means a higher level so maybe you mean `# How to use` here?

**Comment by minhnh** [comment]
I'm not so familiar with `mcr_object_recognition_mean_circle`. Maybe @sthoduka can give better comments on documentation for the package? I do remember that there's a how-to-use README somewhere though, which should probably be included in this as well.

File: mcr_object_recognition_mean_circle/README.md (hunk 1 above)
Comment (sthoduka) [inline mcr_object_recognition_mean_circle/README.md]: Can you change this to: "How to train a model"

File: mcr_object_recognition_mean_circle/README.md [hunk 2]
```diff
@@ -1 +1,32 @@
...
+
+Clone this repository from b-it-bots. Run:
+git clone git@github.com:b-it-bots/python-pcl.git
+
+## Install Python-pcl
+
+Go inside the python-pcl folder and run the following:
+sudo python setup.py install
+
+## Getting the classifier
```
Comment (sthoduka) [inline mcr_object_recognition_mean_circle/README.md]: Training a classifier

File: mcr_object_recognition_mean_circle/README.md [hunk 3]
```diff
@@ -1 +1,32 @@
 The current model is trained with sklearn version 0.14.1.
+
+### How to use
+## Install sklearn
+
+Run:
+pip install sklearn
```
Comment (argenos) [inline mcr_object_recognition_mean_circle/README.md]: Are this really required? our dependencies should be managed with `rosdep`, not manually with pip

**Review by minhnh — CHANGES_REQUESTED** [review]
@1apple7seeds  can you resolve the changes as commented below?

File: mcr_object_recognition_mean_circle/README.md (hunk 3 above)
Comment (minhnh) [reply mcr_object_recognition_mean_circle/README.md]: @sthoduka would it be better here to have a `requirements.txt` that you can refer to in the `README`?

Comment (sthoduka) [reply mcr_object_recognition_mean_circle/README.md]: yep

**Review by minhnh — CHANGES_REQUESTED** [review]
@1apple7seeds can you add a `requirements.txt` file (documentation about what the file does can be found [here](https://pip.pypa.io/en/stable/user_guide/#requirements-files)) and link to it in the README?

**Comment by argenos** [comment]
I changed the base branch to `devel`. See #29

**Review by argenos — CHANGES_REQUESTED** [review]
@mhwasil Please fix the merge conflicts.

**Review by argenos — APPROVED** [review]

**Comment by argenos** [comment]
I would recommend adding the dependencies to the `package.xml` instead of the README so that they are handled by `rosdep`, the packages are available from `apt` after all:
* python-sklearn
* cython
* python-pcl - if you do have commits here, then I would recommend adding it to the `rosinstall` file and running the missing steps in the CI

@sthoduka @deebuls @abhishek098  since this is mostly atwork code, I leave up to you whether you want to add those changes or merge like this

**Review by minhnh — APPROVED** [review]

**Review by deebuls — APPROVED** [review]

**Pull request merged**